articles/validators.go

```diff
@@ -1,22 +1,41 @@
 package articles
 
 import (
+	"fmt"
+	"golang-gin-realworld-example-app/common"
+	"golang-gin-realworld-example-app/users"
+
 	"github.com/gosimple/slug"
-	"github.com/wangzitian0/golang-gin-starter-kit/common"
-	"github.com/wangzitian0/golang-gin-starter-kit/users"
-	"gopkg.in/gin-gonic/gin.v1"
+	// "github.com/wangzitian0/golang-gin-starter-kit/common"
```

Comment:
remove it？

Reply:
Are you asking me to remove commented line Or comment on line?

Reply:
Not need to keep the comment, just remove all no needed package.